templates/model/pos_test.go

```diff
@@ -36,7 +36,7 @@ func TestPosErrorf(t *testing.T) {
 			pos:     &ConfigPos{10, 11},
 			fmtStr:  "Oh no! some number %d: %w",
 			args:    []any{345, fmt.Errorf("wrapped error")},
-			wantErr: "at spec.yaml line 10 column 11: Oh no! some number 345: wrapped error",
```

Comment:
Maybe we need to add file information to ConfigPos. Before, there was only one YAML file so we could assume it was always `spec.yaml`. But now there's a second YAML file so perhaps we need to print its name. What do you think?

This might be very tricky to implement. Our YAML parsing library doesn't make it easy to get this information. So we might want to do it in a followup PR.

Reply:
Agree. As discussed off-line, we'll impl this in following PRs.